ui/src/message_popup/popup_question.jsx

```diff
@@ -58,7 +60,9 @@ export default React.createClass({
   },
 
   updateTimer() {
-    this.setState({ elapsedMs: this.state.elapsedMs + ONE_SECOND });
+    if(!this.state.done){
```

Comment:
I'd clarify this to `isDoneRevising` or something to distinguish from "done with the problem" like `onDone`.